lib/note-detail/style.scss

```diff
@@ -81,6 +81,11 @@
     cursor: inherit;
   }
 
+  // Show pointer cursor on checkboxes
```

Comment:
we probably don't need this comment as the CSS is pretty clear

Reply:
You're right. I'll remove it!